server/utils/createVideosDir.js

```diff
@@ -0,0 +1,11 @@
+const uuid = require('uuid');
+
+module.exports = () => {
+    const videosDir = uuid();
+    while (!fs.existsSync(videosDir)){
+        videosDir = uuid();
```

Comment:
you can not reassign value to a `const` variable

Reply:
bruh